src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import core.basesyntax.service.FileReaderService;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
+    @Override
+    public List<String[]> read(File input) {
+        try (FileReader fileReader = new FileReader(input)) {
+            try (CSVReader reader = new CSVReader(fileReader)) {
```

Comment:
```suggestion
            try (CSVReader reader = new CSVReader(new FileReader(input))) {
```

I think we can use one `try` construction here